### ISSUE TITLE
Support plural words

### ISSUE THREAD

**Issue by codeadict** [ivanseidel/Is-Now-Illegal#12]
Will be nice to have plural support, actually everything says `IS NOW ILEGAL` , if you can make it detect plural words and replace the IS for ARE like "DOUBLE UNDERSCORES ARE NOW ILEGAL".

**Comment by crazy4groovy** [comment]
A Facebook posted link to http://isnowillegal.com/?the%20media didn't work, just the front page showed up.

I think maybe supporting http://isnowillegal.com/?the+media would help?

**Comment by brunolemos** [comment]
[OFF] @crazy4groovy Links on facebook must start with "share.isnow...", like http://share.isnowillegal.com/the%20media

**Comment by qgustavor** [comment]
> ... detect plural words ...

Seems an option would be easier to implement plural word detection: English pluralization rules are inconsistent, so detecting plural forms can fail some cases. An example: [Pluralize.js](https://github.com/blakeembrey/pluralize) fails [in this case](https://www.reddit.com/r/anime/comments/5e2wlw/this_is_the_year_2100_what_do_you_think_the_anime/da9h6mw/?context=3).

*Edit:* in the other hand if you want to make it simple, easy to use (i.e. the user don't needs to select any option, just type the word), and don't care about edge-cases it do the work quite well.

**Comment by pedrofracassi** [comment]
I think adding a checkbox on the page would work perfectly

**Comment by codeadict** [comment]
Yeah I guess you can keep it simple and forget about edge cases.

**Comment by 110Percent** [comment]
I agree with the checkbox idea. It's a surefire way to prevent misrecognition of plurals.

**Comment by CyanideCarrot** [comment]
i think the simplest way is http://isnowillegal.com/?Grammar

**Comment by pedrofracassi** [comment]
lmao
